benchmark/crypto/randomBytes.js

```diff
@@ -5,7 +5,7 @@ const { randomBytes } = require('crypto');
 
 const bench = common.createBenchmark(main, {
   size: [64, 1024, 8192, 512 * 1024],
-  n: [1e3],
+  n: [1e4],
 });
 
 function main({ n, size }) {
```

Comment:
Can you please update the benchmark code to:

```js
// add together with imports
const assert = require('assert');


let _cryptoResult;

function main({ n, size }) {
  bench.start();
  for (let i = 0; i < n; ++i)
    _cryptoResult = randomBytes(size);
  bench.end(n);
  // Avoid V8 deadcode (elimination)
  assert.ok(_cryptoResult);
}
```

Reply:
Done.